app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
The `skills` attribute is defined as a class variable, which means it is shared across all instances of the class. This can lead to unexpected behavior if one instance modifies the list. Consider defining `skills` as an instance variable within the `__init__` method.